server/src/main/java/io/crate/execution/dml/upsert/ShardUpsertRequest.java

```diff
@@ -118,7 +116,6 @@ public ShardUpsertRequest(StreamInput in) throws IOException {
         }
         continueOnError = in.readBoolean();
         duplicateKeyAction = DuplicateKeyAction.values()[in.readVInt()];
-        validateConstraints = in.readBoolean();
```

Comment:
We need to keep reading the boolean for older versions for streaming BWC

Reply:
Thank you!

Reply:
Actually, just realized that I should not remove `validateConstraints` at all, was confused with the disable flag discussed.. Dropping the changeset.